#! /bin/sh /usr/share/dpatch/dpatch-run
## 45-fix-man-references.dpatch by Sven Mueller <debian@incase.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix manpage references to match manpages installed by debian packages

@DPATCH@
diff -urNad trunk/spampd /tmp/dpep.frHYYX/trunk/spampd
--- trunk/spampd	2005-10-18 21:09:01.000000000 +0200
+++ /tmp/dpep.frHYYX/trunk/spampd	2005-10-18 21:09:01.353161334 +0200
@@ -1416,5 +1416,5 @@
 
 =head1 See Also
 
-perl(1), Spam::Assassin(3), L<http://www.spamassassin.org/>, 
-L<http://www.WorldDesign.com/index.cfm/rd/mta/spampd.htm>
+perl(1), Mail::SpamAssassin(3pm), L<http://www.spamassassin.org/>, 
+L<http://www.WorldDesign.com/index.cfm/rd/mta/spampd.htm>, spamassassin(1p)
